.travis.yml

```diff
@@ -1,21 +1,7 @@
 language: android
 android:
   components:
-    # Uncomment the lines below if you want to
-    # use the latest revision of Android SDK Tools
     - tools
     - platform-tools
-    - tools
-
-    # The BuildTools version used by your project
+    - android-26
     - build-tools-26.0.2
-
-    # The SDK version used to compile your project
-    - android-25
-
-    # Additional components
-    - extra-google-google_play_services
```

Comment:
What is the use case for having these? Travis doc states:
"While the following components are preinstalled, the exact list may change without prior notice. To ensure the stability of your build environment, we recommend that you explicitly specify the required components for your project."

Reply:
Yeah good question. My guess is that with the most recent version of the toolchain, the download of gms (and other components) happens automatically and transparently. No more need for these lines (the latest gms version might not even be available anymore from this install). Since the build is successful it means that the dependencies are definitely available.